components/gc-toc/toc-en.html

```diff
@@ -0,0 +1,56 @@
+---
+{
+	"title": "In-page table of contents",
+	"language": "en",
+	"altLangPage": "toc-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-16"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<p>The In-Page Table of Contents Pattern is a list of links that lead to subsections of the same page. This pattern is intended for long documents that exist as a single page.</p>
+<nav role="navigation" class="gc-toc">
+	<h2>On this page</h2>
+	<ul>
+		<li><a href="#section1">First section heading</a></li>
+		<li><a href="#section2">Second section heading</a></li>
+		<li><a href="#section3">Third section heading</a></li>
+	</ul>
```

Comment:
The last heading should be there considering it is an h2 sibling

```suggestion
		<li><a href="#section3">Third section heading</a></li>
		<li><a href="#code">Code</a></li>
	</ul>
```